plugin-bazel-server/build.gradle

```diff
@@ -1,21 +1,42 @@
-
-
 plugins {
-    id 'com.github.rodm.teamcity-server' version '1.4.1'
+    alias libs.plugins.teamcity.server
+    alias libs.plugins.changelog
+}
+
+changelog {
+    path = file("../CHANGELOG.md").canonicalPath
+    groups = ["Added", "Changed", "Fixed"]
 }
 
 teamcity {
-    version = teamcityVersion
+    version = libs.versions.teamcity.get()
 
     server {
-        descriptor = project.file('teamcity-plugin.xml')
-        archiveName = 'teamcity-bazel-plugin'
-        tokens = [Plugin_Version: project.version]
+        descriptor {
+            name = "bazel"
+            displayName = "Bazel build support"
+            description = "Provides build facilities for bazel projects"
+            downloadUrl = "https://github.com/JetBrains/teamcity-bazel-plugin"
+            version = project.version.toString()
+            vendorName = "JetBrains"
+            vendorUrl = "https://www.jetbrains.com/"
+            useSeparateClassloader = true
+            allowRuntimeReload = true
+            minimumBuild = "40000"
```

Comment:
seems we're missing a `node-responsibilities-aware`

Reply:
done